src/care/CMakeLists.txt

```diff
@@ -38,7 +38,6 @@ set(care_headers
     KeyValueSorter_inst.h
     local_host_device_ptr.h
     local_ptr.h
-    LoopFuser.h
```

Comment:
these should always be included.  LoopFuser.h provides implementations for the loop macros that are part of CAREs API for all builds for all configurations.